.dockerignore

```diff
@@ -0,0 +1,23 @@
+**/__pycache__
```

Comment:
You'll need to selectively add these to .gitignore too so that they don't end up on GitHub.